@codex
```
Run pre-commit run --show-diff-on-failure --color=always --all-files
check for added large files..............................................Passed
check for merge conflicts................................................Passed
check yaml...............................................................Passed
fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing .github/dependabot.yml

trim trailing whitespace.................................................Passed
ruff (legacy alias)......................................................Passed
pre-commit hook(s) made changes.
If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
To run `pre-commit` as part of git workflow, use `pre-commit install`.
All changes made by hooks:
diff --git a/.github/dependabot.yml b/.github/dependabot.yml
index 6b8c1e5..444a111 100644
--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -19,4 +19,3 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "deps"
-
Error: Process completed with exit code 1.
```

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -19,4 +19,3 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "deps"
-

--- a/.github/workflows/pre-commit.yml
+++ b/.github/workflows/pre-commit.yml
@@ -20,4 +20,5 @@ jobs:
           python-version: "3.11"
 
       - name: Run pre-commit
-        uses: pre-commit/action@v3.0.1
+        # v3.0.1
+        uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd

